metadata_service/config.py

```diff
@@ -118,37 +103,43 @@ class LocalConfig(LocalGremlinConfig, Config):
     }
 
 
-# The databuilder expects this to be False currently. We are defaulting to true because the testing expects this
-if bool(distutils.util.strtobool(os.environ.get('IGNORE_NEPTUNE_SHARD', 'False'))):
-    shard_set_explicitly('')
+class AtlasConfig(LocalConfig):
```

Comment:
let me know wdyt @verdan

Reply:
sweeet! 💯

Reply:
removed everything except `WATERMARK_DATE_FORMATS` as I think table and db entity names should not be configurable anyways. so it makes Atlas Proxy follow regular configuration in 99%

Reply:
@mgorsk1 let's try to make `WATERMARK_DATE_FORMATS` generic as well for all proxies, and not make it specific to Atlas. This will remove the AtlasConfig altogether.

Reply:
I am not sure if design of databuilder/watermak feature on neo4j side actually requires this, it's very Atlas specific

Reply:
we don't need `WATERMARK_DATE_FORMATS` in the databuilder AFAIK.